### PR TITLE
Add notes on making DID public before creating Schema definition

### PR DESCRIPTION
**Pull request by Selbin**

### CODE REVIEW

File: credential-issue-and-verification-api-user-guide.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ In your browser you can start three tabs to execute the APIs using swagger.
 
 # Schema definition by a legal entity
 On [Test Center Agent](agent1.swagger.localhost), you can execute the schema definiton API to register the schema in the ledger. Ideally this is defined by a legal entity or a standardisation body.
-
+To define schema first you have to create a public DID. To create public DID follow step 1 of [Credenial issuance by the issuer (Test Center)](# Credenial issuance by the issuer (Test Center))
```
Comment (georgepadayatti) [inline credential-issue-and-verification-api-user-guide.md]: @Selbin Can you fix the link to `#credenial-issuance-by-the-issuer-test-center`

**Review by georgepadayatti — CHANGES_REQUESTED** [review]

File: credential-issue-and-verification-api-user-guide.md (hunk 1 above)
Comment (lalc) [reply credential-issue-and-verification-api-user-guide.md]: Noticed that the link to local agent swagger is broken. Please check others as well and fix them.

**Review by lalc — APPROVED** [review]

File: credential-issue-and-verification-api-user-guide.md (hunk 1 above)
Comment (Selbin) [reply credential-issue-and-verification-api-user-guide.md]: ok

Comment (Selbin) [reply credential-issue-and-verification-api-user-guide.md]: fixed those issues

**Review by georgepadayatti — APPROVED** [review]

**Pull request merged**